xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -277,6 +280,8 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
       return -1;
     }
 
+    m_SamplesOut = m_pOutFrame->pts;
```

Comment:
did you log pts? pts of outFrame is independent of input in this case. it should never be equal to AV_NOPTS_VALUE. further why should this behave different on Pi?